.github/workflows/test.yml

```
@@ -9,47 +9,47 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest]
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x, 20.x]
```

```suggestion
        node-version: [14.x, 16.x, 18.x, 20.x]
```

Keep node 14.x support for now